transcom/mymove
# MB-16796 Correct issue where name and date data don't populate on the legal agreement

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16796?atlOrigin=eyJpIjoiMjZlMGQ0NzZlZDljNDAyN2FlODZlMjE1OTVjMWM2YWMiLCJwIjoiaiJ9)

## Summary

This fix should update the Legal PDF to print the customer’s name and the date of the customer’s signing as presented when the customer sets up their shipment and signing. Check out 'How to test' below for details.

If we should limit to certain statuses, it definitely should be more than just submitted. Is there any problem with just not limiting to any status in that conditional?

pdf still has no sign/date (at least before step 5 is done). Is that expected functionality for PPM's pdf?

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the customer app in ephemeral (and local if you'd like too, I didn't notice any difference between them but please let me know if you do).
2. Create a new user with that button and make sure it allows you to see sign/date in the pdf after you've submitted everything
3.  use your login.gov to make move(s) and make sure it allows you to see sign/date in the pdf after you've submitted everything
4. you can also check a lot of the test data users' pdfs of various types of moves too to see if they're signed/dated.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
